package.json

```diff
@@ -136,6 +136,7 @@
     "chokidar": "~2.0.0",
     "commander": "~2.13.0",
     "d3": "~4.12.2",
+    "decode-uri-component": "^0.2.0",
```

Comment:
can you change to ~ instead of ^, even though it is locked it shows differently that way when they push a minor version vs a patch